packages/block/src/header.ts

```diff
@@ -1,10 +1,20 @@
 import Common from 'ethereumjs-common'
-import * as utils from 'ethereumjs-util'
-import { BN } from 'ethereumjs-util'
+import {
+  zeros,
+  KECCAK256_RLP_ARRAY,
+  KECCAK256_RLP,
+  toBuffer,
+  defineProperties,
+  bufferToInt,
+  rlphash,
+} from 'ethereumjs-util'
 import { Blockchain, BlockHeaderData, BufferLike, ChainOptions, PrefixedHexString } from './types'
 import { Buffer } from 'buffer'
 import { Block } from './block'
 
+const { BN } = require('ethereumjs-util')
+import IBN = require('bn.js')
```

Comment:
What's the back story on `BN` vs. `IBN`?

Reply:
I'm not too sure what was going on, I was just trying to resolve some typescript build errors. The error was "Type originates at this import" regarding bn.js in `ethereumjs-util` `externals.d.ts` - might be some confusion because the bn.js types are defined in a `@types` package. I did check and they are in deps and not devDeps for `ethereumjs-util` though so I would have hoped for them to be included in a clean way, but it did not let me simply do `import { BN } from 'ethereumjs-util'`.